spell/spell-crafter-mainnet-workflow.md

```diff
@@ -139,6 +147,9 @@ Repo: https://github.com/makerdao/spells-mainnet
     * [ ] Sanity checks of the constructor arguments
     * [ ] Sanity checks of all values added/updated by the spell function
     * [ ] End-to-end "happy path" interaction with the module
+  * IF bug bounty updates are present
+    * [ ] Test that all bug bounty registry calls execute successfully
+    * [ ] Verify `make update-bug-bounty` returns empty diff in test environment after spell execution
```

Comment:
This specific step should probably be in the checklist right after "cast on tenderly".